docs/install-nix.md

```diff
@@ -0,0 +1,45 @@
+# Install Nix
+
+## Prerequisites
+- Nix
+	- At least version 2.4 >= and < 2.7
+- (Linux with systemd)
+	- You can run Cicero on other platforms but the [Nix actions library](https://github.com/input-output-hk/cicero/blob/main/pkgs/cicero/evaluators/nix/lib.nix) requires [nomad-driver-nix](https://github.com/input-output-hk/nomad-driver-nix).
```

Comment:
Maybe just remove that part. The actions library in the cicero repo will be removed soon, and with tullia you no longer need systemd or nomad-driver-nix.